examples/kitchen-sink/src/app.css

```
@@ -0,0 +1,42 @@
+:root {
+	color-scheme: dark;
+	--bg: #10110f;
+	--fg: #efe6d4;
+	--muted: #9a927f;
+	--accent: #f9b233;
+	--line: #2c2a22;
+	--card: #18170f;
+	--ok: #8fd18a;
+	--bad: #e07a6a;
+	font-family: "Iowan Old Style", "Palatino Linotype", Palatino, Georgia, serif;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)